packages/core/lib/context_utils.js

```diff
@@ -42,11 +42,11 @@ var contextUtils = {
    * @alias module:context_utils.resolveManualSegmentParams
    */
 
-  resolveManualSegmentParams: function resolveManualSegmentParams(params) {
+  resolveManualSegmentParams: function resolveManualSegmentParams (params) {
```

Comment:
Confirmed with Carol,
The linter has not been run in a long time which is causing the large code churn.